pages/spacedeck/[[...roomId]].js

```diff
@@ -209,7 +211,7 @@ export default function Spacedeck() {
                                         key={spacedeckRoomId}
                                         name={room.name}
                                         href={`${spacedeckPath}/${spacedeckRoomId}`}
-                                        selected={roomId === spacedeckRoomId}
+                                        selected={!auth.connectionStatus.spacedeck || roomId === spacedeckRoomId}
```

Comment:
This is not correct. I understand that you're trying to make the link look disabled, however, this attribute is called `selected`. This is confusing to understand later on in life.

Instead we should add a `disabled` attribute to our `<ServiceLink>` component.

Reply:
Other than that I'm asking myself: Do we really still need this?
Why do we care about making a link's color gray if it can never seen under this condition?

If the connection to the Spacedeck server is interrupted, we show the new login prompt component. We will never show the list of Spacedeck spaces in that case.

Reply:
yeah you're right thats now unnecessary with the new view.